docs/meshstack.github.pipeline-automation.md

```diff
@@ -1,17 +1,17 @@
 ---
 id: meshstack.github.pipeline-automation
-title: Pipeline Automation
+title: GitHub Actions Building Blocks
 ---
 
-With meshStack, you can publish automation workflows in other platforms directly to the marketplace, allowing application teams to easily access and initiate automation in a user-friendly, structured format in a central place. By providing triggers for automation in the marketplace, you enable teams to leverage these workflows without needing in-depth Git expertise.
+With meshStack, you can publish your GitHub action workflows directly to the marketplace, allowing application teams to easily access and initiate automation in a user-friendly, structured format in a central place. By providing triggers for automation in the marketplace, you enable teams to leverage these workflows without needing in-depth Git expertise.
 
-Platform engineers can offer "Pipeline Building Blocks" to trigger GitHub Action Workflows directly when added to a tenant. These building blocks can be published to the marketplace, creating a seamless experience for teams to use existing automations.
+Platform engineers can offer "GitHub Actions Building Blocks" that trigger a GitHub Action Workflow. These building blocks can be published to the marketplace, creating a seamless experience for teams to use existing automations.
```

Comment:
above you use GitHub Action Workflow in lower case. As its a product name we should keep the upper case spelling throughout documentation